content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -86,7 +86,7 @@ dimensions.
             {
                 "name": "requests_total",
                 "dimensions": {
-                    "destination_port": "destination.port",
+                    "destination_port": "string(destination.port)",
```

Comment:
Not sure but is this valid conversion in `JSON` ?

Reply:
the conversion will happen on the proxy side.. this is an expression that istio/proxy will handle...